.travis.yml

```diff
@@ -0,0 +1,60 @@
+# Copyright 2018 JanusGraph Python Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+language: python
+
+python:
+- '3.6'
+- '3.5'
+- '3.4'
+- '3.5-dev'
+- '3.6-dev'
+
+env:
+  global:
+    - ENV_NAME=tmpenv
+
+install:
+  - pip install virtualenv
+  # Create Virtual env within which all packages will be installed. Default Python version is Travis' Python version
+  - virtualenv "${ENV_NAME}"
+
+before_script:
+  # Install all pre-requisites for building library and docs
+  - bash before-script.sh "${ENV_NAME}"
+
+script:
+  - bash build-docs.sh "${ENV_NAME}"
+  - bash build-library.sh "${ENV_NAME}"
```

Comment:
I don't see any tests executed in [the Travis build log](https://travis-ci.org/github/JanusGraph/janusgraph-python/jobs/645679201). Are they just not logged for some reason or what's the problem there?

Reply:
![image](https://user-images.githubusercontent.com/1013906/80024355-88cd4380-84fc-11ea-9d92-7b4b30ec9e0d.png)
We see, `2 integration tests passed` and `all unit tests passed` doesn't that verify that the tests ran and got executed? Are we supposed to see some other logs? Though strange is seeing `warning: no unit test executed`

Bellow is snapshot from my Ubuntu WSL
![image](https://user-images.githubusercontent.com/1013906/80024550-dc3f9180-84fc-11ea-9377-56089930d79e.png)
Again similar messages. 

I feel we see `no unit test executed` because, my naming pattern isn't the default pattern which pybuilder expects for unit tests but rather bit different. They expect tests to be `_tests` whereas my files are `_test` and to make them execute, I've added a criteria to pick those type of file suffix.

Reply:
This was a nice catch @FlorianHockmann . Somehow unit tests were getting skipped because I needed to specify the format of unit test files as `*_test` whereas I was using `_test` and missed the asterix. I verified that in recent commit and we can see logs for unit tests coming up in Travis now.